src/utils/sections.ts

```diff
@@ -172,7 +172,7 @@ export const getProgramSections = (): Section[] => {
     return [
         {
             name: 'HeaderOnNow',
-            itemTypes: 'Programs',
+            itemTypes: 'Program',
```

Comment:
here to the other

Reply:
ok ill just make a new getHomeSections with a different type to match whats sent from the api. Or I could combine both types in Section. itemTypes was typed string which I think should just be the string literals instead.

Reply:
you can use 'parametersOptions'

Reply:
oh yea that makes sense. Let me fix it.

Reply:
ok pushed an update. What I was doing was redundant. Thanks